games/Rock Paper Scissors.c

```diff
@@ -0,0 +1,96 @@
+/**
+* @file Rock Paper Scissors.c
+* @author [Aswin P Kumar](https://github.com/AswinPradeep3)
+* @brief [Rock Paper Scissors game](https://en.wikipedia.org/wiki/Rock_paper_scissors)
+* implementation in c
+* @details Rock Paper Scissors is a game in which the user can play with the computer
+* The user has to select an option from rock, paper or scissor.
+* The option of the computer is randomly generated usinng the random()

```

Comment:
An interesting concept would be if there's a more intelligent AI, in hard, easy, and medium modes. Obviously the user should have an opportunity to win, but the AI as well. What do you think?